packages/ui/src/app/pages/Proposals/CurrentProposals.stories.tsx

```diff
@@ -785,7 +785,7 @@ export const SpecificParametersFundingRequest: Story = {
 
 export const SpecificParametersMultipleFundingRequest: Story = {
   play: specificParametersTest('Funding Request', async ({ args, createProposal, modal, step }) => {
-    const bob = member('bob')
+    const dave = member('dave')
```

Comment:
Are these changes necessary ? It will conflict with #4713 when `dev` will be merged into `validators` (which I should have done earlier already but I forgot :facepalm:).

Reply:
I removed this change.